src/libraries/System.Private.Xml/tests/Xslt/TestFiles/TestData/xsltc/baseline/bft35.txt

```diff
@@ -1,5 +1,5 @@
-Microsoft (R) XSLT Compiler version 2.0.61016
-for Microsoft (R) Windows (R) 2005 Framework version 2.0.50727
-Copyright (C) Microsoft Corporation 2007. All rights reserved.
 
-fatal error : Source file 'nul.xsl' could not be found. ---> Win32 device names are not allowed.
+
+
+
+.*error : Source file 'nul\.xsl' could not be found\..*
```

Comment:
This is the only baseline file changed other than by regex-escaping content and removing logo banner lines.